SPARKCONF.md

```diff
@@ -0,0 +1,155 @@
+##Spark Configuration
+
+oni-ml main component uses Spark and Spark SQL to analyze network events and produce a list of least probable events
+or most suspicious. 
+
+Before running oni-ml users need to configure the component. Here are our recommended settings.  
+
+### Yarn tuning
```

Comment:
Perhaps a title on "General Yarn Tuning" as this section seems to be a pointer to resources on tuning Yarn and Spark-on-Yarn for general purposes.